D2L.Hypermedia.Siren/SirenField.cs

```diff
@@ -79,7 +100,7 @@ public class SirenField : ISirenField {
 			bool name = m_name == other.Name;
 			bool @class = m_class.OrderBy( x => x ).SequenceEqual( other.Class.OrderBy( x => x ) );
 			bool type = m_type == other.Type;
-			bool value = m_value == other.Value || m_value != null && m_value.Equals( other.Value );
+			bool value = m_value == other.Value || ( m_value != null && m_value.Equals( other.Value ) );

```

Comment:
Not necessary to add these parentheses

Reply:
VS auto format did it, so I believe this puts it more in line with our coding style.